src/commons/assessmentWorkspace/__tests__/AssessmentWorkspace.tsx

```diff
@@ -24,7 +24,6 @@ const defaultProps: AssessmentWorkspaceProps = {
   handleAssessmentFetch: (assessmentId: number) => {},
   handleBrowseHistoryDown: () => {},
   handleBrowseHistoryUp: () => {},
-  handleChapterSelect: (chapter: any, changeEvent: any) => {},
   handleClearContext: (library: Library) => {},
```

Comment:
Probably should update the default prop of `handleClearContext` to include the init boolean here.